docs/deep-dives/esm.md

```diff
@@ -255,7 +259,52 @@ module.exports = {
 
 [`'...'` syntax]: https://webpack.js.org/configuration/resolve/#resolveconditionnames
 
-### 3. Switch to Vitest and ESM
+### 3. Switch to tsdown
+
+Within `build-package` we will switch to calling [`tsdown`](https://tsdown.dev/) instead of wrapping the TypeScript compiler directly. This will allow us to continue publishing both CJS and ESM after we've migrated to ESM.
```

Comment:
```suggestion
`skuba build-package` will switch to calling [`tsdown`](https://tsdown.dev/) instead of wrapping the TypeScript compiler directly. This will allow us to continue publishing both CJS and ESM after we've migrated to ESM.
```